src/Illuminate/Database/Query/Builder.php

```diff
@@ -599,7 +599,7 @@ public function crossJoin($table, $first = null, $operator = null, $second = nul
     /**
      * Get a new join clause.
      *
-     * @param  \Illuminate\Database\Query\Builder  $parentQuery
+     * @param  self  $parentQuery
```

Comment:
Please don't change this. It is a stylistic choice to use the full name in the phpdoc.